packages/@okta/vuepress-site/docs/api/resources/zones/index.md

```diff
@@ -2,23 +2,160 @@
 title: Zones
 category: management
 ---
-
 # Zones API
 
 > This API is an <ApiLifecycle access="ea" /> feature.
 
 The Okta Zones API provides operations to manage zones in your organization. Zones may be used to guide policy decisions.
 
-IP zones are the only type currently supported by the Zones API. IP zones are used to group IP address ranges so that policy decisions can be made based on the client's IP location.
+## Zone Model
+
+### Base Network Zone Properties
+
+The Base Network Zone object defines the base properties which concrete Network Zone types (IP and Dynamic) inherit.
+
+All Network Zone objects define the following attributes:
+
+| Field Name     | Description                                                                                 | Data Type                                     | Required        | Max Length    |
+| :------------- | :------------------------------------------------------------------------------------------ | :-------------------------------------------- | :-------------- | :------------ |
+| type           | Type of zone is one of the following: `IP`, `DYNAMIC`                                              | String                                        | Yes             | N/A           |
+| id             | Unique identifier for this zone                                                             | String                                        | No (Assigned)   | N/A           |
+| name           | Unique name for this zone                                                                   | String                                        | Yes             | 128 (chars)   |
+| system           | Indicates if this is a system network zone. For admin-created zones, this is always `false`                                                                 | boolean                                        | No  (Assigned)           | N/A   |
+
+### IP Zone Properties
+
+The follow attributes are defined by IP Zone objects:
+
+| Field Name     | Description                                                                                 | Data Type                                     | Required        | Max Length    |
+| :------------- | :------------------------------------------------------------------------------------------ | :-------------------------------------------- | :-------------- | :------------ |
+| gateways       | IP addresses (range or CIDR form) of this zone                                              | Array of [Address Objects](#address-object)   | No              | 150 (entries) |
+| proxies        | IP addresses (range or CIDR form) allowed to forward request from gateway addresses above   | Array of [Address Objects](#address-object)   | No              | 150 (entries) |
```

Comment:
Please also doc that these are the proxies the customer trusts to forward requests to Okta. All the proxies defined in all the IP zones for the orgs are automatically trusted for threat insights and also for identifying the client ip while resolving the geo-location, proxy type, asn.